demos/projects/ESPRESSIF/aziotkit/components/sample-azure-iot/Kconfig.projbuild

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+menu "Azure IoT middleware for FreeRTOS Main Task Configuration"
+

```

Comment:
The configuration of this sample excludes the `AZURE_SAMPLE_USE_PLUG_AND_PLAY` config, since this sample works only with `sample_azure_iot_pnp`